foundation/org.eclipse.persistence.core.test.framework/src/main/java/org/eclipse/persistence/testing/tests/performance/emulateddb/EmulatedConnection.java

```diff
@@ -327,10 +328,10 @@ public boolean isClosed() throws SQLException {
      */
     @Override
     public DatabaseMetaData getMetaData() throws SQLException {
-        if (connection != null) {
-            return connection.getMetaData();
```

Comment:
if there is an existing connection, metadata should be taken from there; in the other words, this piece should have stayed